pkgs/top-level/haskell-packages.nix

```diff
@@ -397,6 +401,25 @@ in {
       llvmPackages = pkgs.llvmPackages_15;
     };
     ghc96 = compiler.ghc963;
+    ghc981 = callPackage ../development/compilers/ghc/9.8.1.nix {
+      bootPkgs =
+        # For GHC 9.2 no armv7l bindists are available.
```

Comment:
I copied this from the definition for GHC 9.6 above, not sure if it needs to change.

Reply:
The bootstrap compiler probably needs to be a binary package? When copying the existing `*-binary.nix` files, beware of https://github.com/NixOS/nixpkgs/pull/260410.

Reply:
Thanks, updated.